src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -27,7 +27,21 @@
             <div class='display-icon pull-right'>
                 <span class="glyphicon ${ isShowingResponses ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down'} pull-right"></span>
             </div>
-            
+            <form method="post" action="<%=Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_RESULTS_DOWNLOAD%>">
+                <div id="DownloadQuestion">
```

Comment:
I would suggest that u concatenate ${questionPanel.question.questionNumber} to all your id (DownloadQuestoin, button_download) for easier testing and future javascript might need the id as well. (I think that is the convention in all the tag file as well, looking at id="seeMore-${questionPanel.question.questionNumber}"  in the same div )